### ISSUE TITLE
update RFC 7159 references to RFC 8259

### ISSUE THREAD

**Issue by dret** [w3c/dnt#79]
there's probably nothing else to do other than updating the reference to the latest JSON RFC.

**Comment by dwsinger** [comment]
agreed, this is editorial and we should do it. https://tools.ietf.org/html/rfc7159 Obsoleted by: 8259

**Comment by dret** [comment]
On 2018-06-25 09:09, David Singer wrote:
> agreed, this is editorial and we should do it. 
> https://tools.ietf.org/html/rfc7159 Obsoleted by: 8259

i had a brief look at the repo structure but i had a hard time 
understanding how the repo is organized, so i'll leave creating a PR to 
a more adventurous soul....
